engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/bv/DecimalMinValidatorForNumber.java

```diff
@@ -16,7 +16,7 @@
 import org.hibernate.validator.internal.util.logging.LoggerFactory;
 
 /**
- * Check that the number being validated is less than or equal to the maximum
+ * Check that the number being validated is greater than or equal to the minimum
```

Comment:
While looking at other validators and docs I realized that this was most likely a copy&paste error.